spec/ruby/core/io/write_spec.rb

```diff
@@ -22,7 +22,7 @@
 
   # TODO: impl detail? discuss this with matz. This spec is useless. - rdavis
   # I agree. I've marked it not compliant on macruby, as we don't buffer input. -pthomson
```

Comment:
While you're working on this file, nuke these comments, they're pointless to keep around.